doc/user_guide/Server_Deployment.md

```diff
@@ -209,6 +209,47 @@ Finally, to deploy a Panel app to App Engine run `gcloud app create` and `gcloud
 
 For detailed information and steps, check out this [example](https://towardsdatascience.com/deploy-a-python-visualization-panel-app-to-google-cloud-cafe558fe787?sk=98a75bd79e98cba241cc6711e6fc5be5) on how to deploy a Panel app to App Engine and this [example](https://towardsdatascience.com/deploy-a-python-visualization-panel-app-to-google-cloud-ii-416e487b44eb?sk=aac35055957ba95641a6947bbb436410) on how to deploy a Panel app to Cloud Run.
 
+#### Hugging Face
+
+The guides below assumes you have already signed up and logged into your account at [huggingface.co](https://huggingface.co/).
+
+#### Duplicate an existing space
+
+The easiest way to get started is to  [search](https://huggingface.co/spaces), find and duplicate an existing space. A simple space is
+[MarcSkovMadsen/awesome-panel](https://huggingface.co/spaces/MarcSkovMadsen/awesome-panel).
```

Comment:
@philippjfr . Would you like Panel to setup and maintain a simple template space instead?

Reply:
Yes, definitely.